src/controller/python/chip/yaml/parser.py

```diff
@@ -29,11 +29,170 @@
 from chip.yaml.errors import ParsingError, UnexpectedParsingError
 from .data_model_lookup import *
 import chip.yaml.format_converter as Converter
+from .response_storage import ResponseStorage
 
 _SUCCESS_STATUS_CODE = "SUCCESS"
+_NODE_ID_DEFAULT = 0x12345
+_ENDPOINT_DETAULT = ''  # TODO why is this an empty string
+_CLUSTER_DEFAULT = ''
+_TIMEOUT_DEFAULT = 90
```

Comment:
Seems odd that we enforce these very arbitrary defaults. Perhaps we can make the values mandatory in all YAML files and cause a YAML config failure if missing. This may work temporarily. but we need a TODO not to have implied defaults.

Reply:
For the timeout - this is not something that Terence is adding - this is present in the current yaml parser. I'm not sure there is a nodeid default in the current version, but actually, the nodeid default doesn't actually matter, since, despite the name, in the current impl you run commands against the node id you supplied to the WaitForCommissionee command, which normally uses this nodeid value, but isn't required to.